packages/modules/viewer-react/src/components/iModel/IModelLoader.tsx

```diff
@@ -44,13 +44,16 @@ const IModelLoader = React.memo((viewerProps: ModelLoaderProps) => {
     backstageItems,
     loadingComponent,
   } = viewerProps;
+  const backstageItemsProvided = UiItemsManager.getBackstageItems();
```

Comment:
thinking about it, i could be wrong so please verify, but this is populated by the uiproviders right?

what if we constructed an instance of BackstageItemsProvider ourselves from backstageItems,

then pass that provider along with the array we pass to the useUiProviders hook

from there would we be able to clean up the IModelViewer component?